### PR TITLE
add a simple test for min profile

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

File: tests/util/helm_utils.go [hunk 1]
```diff
@@ -33,15 +33,25 @@ func HelmDepUpdate(chartDir string) error {
 }
 
 // HelmInstallDryRun helm install dry run from a chart for a given namespace
-func HelmInstallDryRun(chartDir, chartName, namespace, setValue string) error {
-	_, err := Shell("helm install --dry-run --debug %s --name %s --namespace %s %s", chartDir, chartName, namespace, setValue)
+func HelmInstallDryRun(chartDir, chartName, namespace, valueFile, setValue string) error {
+	var err error
+	if valueFile != "" {
+		_, err = Shell("helm install --dry-run --debug %s --name %s --values %s --namespace %s %s", chartDir, chartName, valueFile, namespace, setValue)
```
Comment (hklai) [inline tests/util/helm_utils.go]: may want to construct this dynamically? adding more options later will multiply the combinations.

**Comment by linsun** [comment]
@hklai thanks, yes, looking at adding a test case in test-infra now.  What is the sequence?  should the PR in istio go in first then the test infra?

**Comment by hklai** [comment]
@linsun you can add to test-infra first. (yes it will fail for everyone else with a file not found error until you merge this). It will at least let you test this out first.

**Comment by linsun** [comment]
@hklai I added the test to post submit as I don't want to lengthen everyone's pre-submit.  Does this mean I won't see the test result till the PR is good to go and merged?

**Comment by hklai** [comment]
@linsun the test is failing.
https://storage.googleapis.com/istio-prow/logs/e2e-simpleTestsMinProfile/14/build-log.txt

It looks like there is a typo in the filename.

**Comment by linsun** [comment]
@hklai I checked, it was spelled correctly.  Is it possible there is other change needed to copy the file over to the correct dir?

**Comment by linsun** [comment]
@hklai thank you much!!

**Pull request merged**